cpmonitor/tests/cleanupimages_test.py

```diff
@@ -0,0 +1,128 @@
+import os
+import shutil
+import tempfile
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from cpmonitor.models import City
+
+
+class CleanupImagesTest(TestCase):
+    base_path = None
+    images_path = None
+    uploads_path = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.base_path = tempfile.mkdtemp()
+        cls.images_path = os.path.join(cls.base_path, "images")
+        cls.uploads_path = os.path.join(cls.images_path, "uploads")
+
+        cls.env_patcher = patch.dict(
+            os.environ,
+            {
+                "BASE_PATH": cls.base_path,
+                "IMAGES_PATH": cls.images_path,
+                "UPLOADS_PATH": cls.uploads_path,
+            },
+        )
+        cls.env_patcher.start()
+
+        super().setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+
+        shutil.rmtree(cls.base_path, True)
+
+        cls.env_patcher.stop()
+
+    def setUp(self):
+        super().setUp()
+
+        os.mkdir(self.images_path)
+        os.mkdir(self.uploads_path)
+
+        self.assertTrue(os.path.isdir(self.base_path))
```

Comment:
I think we do not need all those assertions in the setup and tear down if we start from a fresh state.

Reply:
Wanted to implement it in a defensive way but I'm fine with it to leave the assertions out